Sources/PackageGraph/PackageGraphRoot.swift

```diff
@@ -133,6 +134,9 @@ extension PackageDescription4.Package.Dependency.Requirement {
 
         case .exactItem(let version):
             return .versionSet(.exact(Version(pdVersion: version)))
+
+        case .localPackageItem:
```

Comment:
I'm still confused as to why we're representing local packages through `Requirement`. Is this the only solution?

Reply:
Requirement is correct though as it *is* a requirement.